tests/python/relay/test_well_formed.py

```diff
@@ -0,0 +1,12 @@
+import tvm
+from tvm import relay
+from tvm.relay.ir_pass import well_formed
+
```

Comment:
try to add a few more cases. for example the case which weak check will pass but strong check won't